test/testRNode.js

```diff
@@ -1,139 +1,144 @@
-const Suite = require('testjs');
-const grpc = require('grpc');
-
-const { RNode, RHOCore, b2h } = require('../rnodeAPI');
+/* global require, module, exports */
```

Comment:
Did you use git to merge my changes in? The **blame** view has all this code attributed to you. Maybe I can fix that later, but I'd rather avoid that sort of thing.

Reply:
Previously copy-pasted this specific change.  Now I have merged in your changes on "live-tests", so this should reflect more accurately.